tests/inheritance/03-override-order.t

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
My only concern is whether we might prefer to start a unit test of `replicate` using pytest. But this works.

Reply:
Fair point, unit tests are cheaper.